### ISSUE TITLE
BSup calculation checking error

### ISSUE THREAD

**Issue by jan-petr** [ExploreASL/ExploreASL#405]
Fix error in the xASL_quant_BSupCalculation function that calculates the percentage of remaining static signal after Background Suppression. In this function, it is checked if the signal readout starts only after the last background suppression pulse is played. This checking didn't really work if both the ReadouTime and the BackgroundSuppressionTime values were vectors and their size differed. To fix it, we now compare if the latest Background suppression pulse is played earlier in time than the earliest readout is started.

### Release summary
Fixing an error in BSup-M0 correction that was probably introduced in this release. If this is the case, then we do not have to report this as an separate issue, but rather assign to another issue that modified this function. Otherwise - if the error is there from release 1.3.0, then write: "Fixing minor error in Background Suppression timing calculation".

**Comment by HenkMutsaerts** [comment]
This explanation:

>> For function xASL_quant_BSupCalculation that calculates the remaining singal after BSup: The check if readout appears only after the last background suppression, there was an error in the calculation when different vector lengths were provided. To fix it - longest Background suppression is now compared with the earliest readout.

Is abracadabra to me. What does "longest background suppression" mean? What is "earliest readout"? :)

**Comment by jan-petr** [comment]
:) Better now?

**Comment by HenkMutsaerts** [comment]
Yep ;)
